### DISCUSSION TITLE
Can not get more logs of DAGS on UI when deploy on Kubernetes

### DISCUSSION THREAD

**Discussion by chuxiangfeng** [apache/airflow#31053] [Q&A]
After update airflow to 2.6.0 and apache-airflow-providers-cncf-kubernetes==6.1.0，Can not get more logs of DAGS on UI when deploy on Kubernetes，It was like suspended animation, but it was able to get the log in the end.
Before update，Kubernetes tasks logs will look like this in UI:
```
{pod_manager.py:285} WARNING - Pod not yet started: t-crawler-gem-ipo-request-71zd2ngd
{pod_manager.py:285} WARNING - Pod not yet started: t-crawler-gem-ipo-request-71zd2ngd
.......
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
........
{pod.py:724} INFO - Skipping deleting pod: t-crawler-gem-ipo-request-71zd2ngd
{taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=d_crawler_gem_ipo, task_id=t_crawler_gem_ipo_request, execution_date=20230504T083622, start_date=20230504T083623, end_date=20230504T084155
{local_task_job_runner.py:232} INFO - Task exited with return code 0
{taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
```
**But now is likes this:**
```
{pod_manager.py:285} WARNING - Pod not yet started: t-crawler-gem-ipo-request-71zd2ngd
{pod_manager.py:285} WARNING - Pod not yet started: t-crawler-gem-ipo-request-71zd2ngd
```
.......
**(In this stage, there is no Pod startup log and no subsequent print during the Running period, so the Pod Running log cannot be recognized, In fact, Pod is running normally on k8s.  f12 of Chrome has been calling the rotation training query log, but no updated log content can be obtained. At the end of the Pod it prints out _Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running_, In theory, the print _Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running_)**
```
ERROR - Error parsing timestamp (no timestamp in message ''). Will continue execution but won't update timestamp
{pod_manager.py:342} INFO - 
{pod_manager.py:397} INFO - Pod t-crawler-gem-ipo-request-71zd2ngd has phase Running
```

**Comment by potiuk** [comment]
This is likely an error in 2.6.0 that should be fixed in 2.6.1 hopefully. PR here: https://github.com/apache/airflow/pull/31101

**Comment by potiuk** [comment]
In the meantime solution is to enable log persistency in the Helm Chart of ours.
